auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -1107,10 +1107,23 @@ public Intent build() {
      * Builder for the intent to start the user authentication flow.
      */
     public final class SignInIntentBuilder extends AuthIntentBuilder<SignInIntentBuilder> {
+
+        private boolean mEnableAnonymousUpgrade;
+
         private SignInIntentBuilder() {
             super();
         }
 
+        /**
+         * Enables or disables upgrading anonymous accounts to full accounts during the sign-in
+         * flow. This is disabled by default.
+         */
+        @NonNull
+        public SignInIntentBuilder setAutoUpgradeAnonymousUsers() {
```

Comment:
This should take in a boolean `autoUpgrade` parameter.

Reply:
Sure but when would they ever call this with false? It's disabled by default.

Reply:
Oh, gotya. Then the javadoc needs to be updated to just `Enables upgrading ...` and the method needs to be renamed to something like `enableAutoUpgradeAnonymousUsers`.

Reply:
Yeah I'm going to ask if I can change the API actually. It'll probably have to take in a boolean to be the same as iOS/Web

Reply:
@lsirac you can definitely change the API to take a boolean without asking anyone.